ObjCTraining/ActivityFeedViewController.m

```diff
@@ -0,0 +1,132 @@
+//
+//  ActivityFeedViewController.m
+//  ObjCTraining
+//
+//  Created by Sunny on 04/04/16.
+//  Copyright © 2016 RWuser. All rights reserved.
+//
+
+#import "ActivityFeedViewController.h"
+#import "ActivityFeedTableViewCell.h"
+#import "ActivityFeedTableViewCell+ConfigureForFeed.h"
+#import "LCFeed.h"
+
+static NSString *const activityFeedCellIdentifier = @"ActivityFeedTableViewCell";
+static NSString *const sectionHeaderDateFormat = @"MMMM d, yyyy";
+static int const cellHeight = 70;
+static int const sectionHeaderHeight = 46;
+
+@interface ActivityFeedViewController ()
+@property NSMutableArray *feeds;
+@end
+
+@implementation ActivityFeedViewController
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.tableView.estimatedRowHeight = cellHeight;
+    [self.tableView registerNib:[UINib nibWithNibName:@"ActivityFeedTableViewCell" bundle:nil] forCellReuseIdentifier:activityFeedCellIdentifier];
+    [self initDataSource];
+    self.tableView.dataSource = self;
```

Comment:
you can set datasource and delegate with interface builder. It's a good idea if you have one table on vc